.gitignore

```diff
@@ -0,0 +1,20 @@
+# Created by https://www.toptal.com/developers/gitignore/api/react
```

Comment:
You should also remove `node_modules` from your repo.

Reply:
Fixed